### DISCUSSION TITLE
What parameters can be considered in the software when Cmax is underestimated?

### DISCUSSION THREAD

**Discussion by Chelton919** [Open-Systems-Pharmacology/Forum#1736] [Questions & Problems]
Dear open-systems-pharmacology community,
  I am developing an adult PBPK model of a small molecule drug, and the Cmax is generally underestimated. The drug has a log P of 0.7 in DrugBank and 1 in the FDA review report. I personally changed log P to 0.2 without strong evidence, the curve fits well. I tried changing the plasma protein binding, but had the same problem as when I changed the log P (outside the range reported in the literature or database). Afterwards I changed the permeability, but since the drug is highly permeable, even increasing the permeability had no effect on the curve. I would like to ask if there are any more parameters that I am overlooking or if I need a reasonable explanation when a parameter is out of the reported range. I don't think this is a large error because there may be errors in experimental values. 
  
![6350a81f565cabf2fd5c9dc42f735b7](https://github.com/Open-Systems-Pharmacology/Forum/assets/116017874/ee7de3db-6edb-43b2-84c2-4cdb6ef0f2f3)
![4455031af8744d4977f8c3036713f31](https://github.com/Open-Systems-Pharmacology/Forum/assets/116017874/0c60dbc6-3aff-4517-b87d-60a0b7e45a16)
Thank you all very much!

**Comment by Yuri05** [comment]
is it an IV administration?

**Reply by Chelton919** [reply]
Hello.
It's not an IV administration, it's an oral tablet. However, the drug is rapidly and completely absorbed when taken orally, with bioavailability approaching 100%. The tablets, oral solution, and injection can be used interchangeably. I used the Weibull function to describe the release feature of the drug. The dissolution time is 15 min.

**Reply by Yuri05** [reply]
* Even though I don't think it would help in this case: can you increase the initial simulation resolution (e.g. set it to 100 pts/h)
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/c07ceda8-55a3-463b-8801-3f8bf38de92c)
* How does your fraction absorbed looks like?
* What happens if you replace the tablet with 15-min "Infusion" into portal vein plasma?
